assets/css/styles.css

```diff
@@ -0,0 +1,322 @@
+/* CSS Resets */
+* {
+    margin: 0;
+    box-sizing: border-box;
+}
+
+/* Color variables */
+:root {
+    --black: #000000;
+    --darkGrey: #333333;
+    --light: #f5f5f7;
+    --red: #E50914;
+}
+
```

Comment:
Not a bad thing, I just really like root variables so great job impletmenting this!